src/pages/Translate_mobile.js

```
@@ -0,0 +1,815 @@
+    setModalShow(true);
+    let translation = {};
+
+    if (!lang && !labelName) {
+      const fileTranslations = translations[filename];
+      if (fileTranslations) {
+        labelName = Object.keys(fileTranslations)[0];
+        lang = Object.keys(fileTranslations[labelName])[0];
+      }
+    }
+
+    if (translations[filename] && translations[filename][labelName]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)